src/components/client/campaign-application/CampaignApplicationForm.tsx

```diff
@@ -51,9 +40,27 @@ const steps: StepType[] = [
   },
 ]
 
-export default function CampaignApplicationForm() {
+type Props = {
+  person: PersonResponse
+}
+
+export default function CampaignApplicationForm({ person }: Props) {
   const [activeStep, setActiveStep] = useState<Steps>(Steps.ORGANIZER)
 
+  const initialValues: CampaignApplicationFormData = useMemo(
```

Comment:
useMemo isn't really needed for this case
 We don't do any calculations here.

Reply:
Okay, removed